### ISSUE TITLE
Create an Indexes Collection for the DB

### ISSUE THREAD

**Issue by torch2424** [Kondeo/ccra-book#2]
We need to write a new java program that will take text input and parse the index that has been exported as plain text such as:

And also such as:

Rule 3.61	Court fees and costs waived by initial application	25
Rule 8.23	Sanctions to compel compliance	25
Rule 8.45	General Provisions	25-27
Rule 8.46	Sealed records	28-31
Rule 8.47	Confidential records	31-34

@julianpoy Also, if you notcie the indexes have their own indexes. I'm guessing you want a more specific index correct?

this will have the following columns:

_id, Title (or law# if same, else law# has it's own column), page#

**Comment by julianpoy** [comment]
_id, title, pageNum
Nomatter if law has number and/or title, it all goes in Title
Yes, I want specific indexes.

**Comment by torch2424** [comment]
This is on hold because of the indexes being very extensive.

**Comment by torch2424** [comment]
@julianpoy now that you have talked to the CCRA what is the status of this issue

**Comment by julianpoy** [comment]
Still on hold. More important things first like #3

**Comment by torch2424** [comment]
I suggest we Do our indexes as such

Homepage:

"Welcome to the app we offer two books and can be used for ccra..."
-Book One
-BookTwo


After you select Book:

-Introduction
- Index 1 first page (The first page after an index of 4 is placed, so if the index is laws about reporting, and contains 3 pages of the titles of all the main laws, the first page AFTER this index)
-Index 2 first page
...
...
...

**Comment by julianpoy** [comment]
Notes from discussion earlier: Once the editpage is done, we can just edit the existing indexes to look better through the graphical editor we ourselves built.
Note from me, now: We dont have to do too much fixing however, since Steve (from CCRA) will fix the indexes. I just want the formatting to look a little better. The actual page number fixes will be done by him. We do however need to hotlink them.

**Comment by torch2424** [comment]
What is the status of this? I am quite confused on the issue

**Comment by julianpoy** [comment]
We just need to go into the index html for each of the indexes and make it a little less broken. The actual page numbers and such will be fixed by Steve from CCRA.

**Comment by julianpoy** [comment]
Very weird tag update there ^

**Comment by julianpoy** [comment]
As of now, this is going to be on the "backburner". I do not believe we are implementing this at all.

**Comment by julianpoy** [comment]
I spoke to Steve a long time ago, and he specified that there would be no index anymore with the search engine.
